testbed/tests/trace_test.go

```diff
@@ -131,42 +131,27 @@ func TestTraceNoBackend10kSPS(t *testing.T) {
 		{
 			Name:                "NoMemoryLimit",
 			Processor:           noLimitProcessors,
-			ExpectedMaxRAM:      200,
-			ExpectedMinFinalRAM: 30,
+			ExpectedMaxRAM:      150,
+			ExpectedMinFinalRAM: 100,
 		},
 		{
 			Name:                "MemoryLimit",
 			Processor:           limitProcessors,
-			ExpectedMaxRAM:      60,
-			ExpectedMinFinalRAM: 10,
+			ExpectedMaxRAM:      70,
+			ExpectedMinFinalRAM: 40,
 		},
 	}
 
 	var testSenders = []struct {
 		name          string
 		sender        testbed.DataSender
 		receiver      testbed.DataReceiver
-		resourceSpec  testbed.ResourceSpec
 		configuration []processorConfig
 	}{
 		{
-			"JaegerGRPC",
-			testbed.NewJaegerGRPCDataSender(testbed.DefaultHost, testbed.DefaultJaegerPort),
-			testbed.NewOCDataReceiver(testbed.DefaultOCPort),
-			testbed.ResourceSpec{
-				ExpectedMaxCPU: 70,
-				ExpectedMaxRAM: 198,
-			},
-			processorsConfig,
-		},
-		{
-			"Zipkin",
-			testbed.NewZipkinDataSender(testbed.DefaultHost, testbed.DefaultZipkinAddressPort),
-			testbed.NewOCDataReceiver(testbed.DefaultOCPort),
-			testbed.ResourceSpec{
-				ExpectedMaxCPU: 120,
-				ExpectedMaxRAM: 198,
-			},
```

Comment:
Why do you remove the jaeger and zipkin tests?

Reply:
They are not needed. We are not testing protocols here, we are testing memory limiter and testing using one protocol is sufficient, the behavior of memory limiter is not protocol dependent.
I cleaned this up more, PTAL.